sim/druid/tank/tank.go

```diff
@@ -72,12 +76,16 @@ func (bear *FeralTankDruid) GetDruid() *druid.Druid {
 
 func (bear *FeralTankDruid) Initialize() {
 	bear.Druid.Initialize()
-	bear.RegisterFeralTankSpells()
+	queuedRealismICD := &core.Cooldown{
+		Timer:    bear.NewTimer(),
+		Duration: core.SpellBatchWindow * 10,
+	}
```

Comment:
❓ Since this is only use for Maul should we just initialize this inside of the Maul code?